Sources/core/objects/Tables.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "Movable.h"
+
+class Tables: public IMovable
+{
```

Comment:
This class describes one piece of table, so it is better to name it just `Table`.